src_c/alphablit.c

```diff
@@ -310,7 +310,7 @@ SoftBlitPyGame(SDL_Surface *src, SDL_Rect *srcrect, SDL_Surface *dst,
                     break;
                 }
                 case PYGAME_BLEND_SUB: {
-#if !defined(__EMSCRIPTEN__)
+#if !defined(__EMSCRIPTEN__) && !defined(_M_ARM64)
```

Comment:
I feel these checks can be removed.

In theory (untested), the neon path (neon being ARM SIMD) should also work on ARM64 windows.

In this same file, we have
```c
#if !defined(PG_ENABLE_ARM_NEON) && defined(__aarch64__)
// arm64 has neon optimisations enabled by default, even when fpu=neon is not
// passed
#define PG_ENABLE_ARM_NEON 1
#endif
```
somewhere at the top. I doubt `__aarch64__` is defined on MSVC, so I propose that this should become
```c
#if !defined(PG_ENABLE_ARM_NEON) && (defined(__aarch64__) || defined(_M_ARM64))
// arm64 has neon optimisations enabled by default, even when fpu=neon is not
// passed
#define PG_ENABLE_ARM_NEON 1
#endif
```

Reply:
https://github.com/DLTcollab/sse2neon/issues/384
Sse2neon does not support MSVC.